sanssouci/lambda_calibration.py

```diff
@@ -80,25 +87,32 @@ def get_perm_p(X, categ, B=100, row_test_fun=stats.ttest_ind):
 
 
 def get_pivotal_stats(p0, t_inv=t_inv_linear, K=-1):
-    """
-    Get pivotal statistic
-
-    Inputs:
-    - p0:  A numpy array of size [B,p] of null p-values obtained from
-           B permutations for p hypotheses.
-    - t_inv: A function with the same I/O as t_inv_linear
-    -  K:  For JER control over 1:K, i.e. joint control of all k-FWER, k<= K.
-           Automatically set to p if its input value is < 0.
-
-    Returns:
-     - A numpy array of of size [B]  containing the pivotal statitics,
-       whose j-th entry corresponds to \psi(g_j.X) with notation of
-       the AoS 2020 paper cited below (section 4.5)
-
-     Reference:
-     - Blanchard, G., Neuvial, P., & Roquain, E. (2020). Post hoc confidence
-       bounds on false positives using reference families.
-       Annals of Statistics, 48(3), 1281-1303.
+    r"""Get pivotal statistic
+
+    Parameters
+    ----------
+    p0 :  array-like of shape (B, p)
+        A numpy array of size [B,p] of null p-values obtained from
+        B permutations for p hypotheses.
+    t_inv : function, default=t_inv_linear
```

Comment:
```suggestion
    t_inv : function
```
(the default comes at the end)